jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/controller/ResourceApiController.java

```diff
@@ -126,6 +153,21 @@ private T readEntityFromPayload(String body) throws JsonProcessingException {
     }
   }
 
+  private List<PatchOperation<T>> readEntitiesFromPayload(List<Object> body) {
+    try {
+      var opList = new ArrayList<PatchOperation<T>>();
+      var patchOperationType = objectMapper.getTypeFactory().constructParametricType(
+          PatchOperation.class,
+          entityType);
+      for (Object ob : body) {
```

Comment:
nit: Could simplify the for loop to a one line statement using stream.
`body.stream().forEach(ob -> opList.add(objectMapper.convertValue(ob, patchOperationType)));`

Reply:
updated :)